Sources/ApolloWebSocket/Compression.swift

```diff
@@ -0,0 +1,183 @@
+//  Created by Joseph Ross on 7/16/14.
+//  Copyright © 2017 Joseph Ross.
+//  Modified by Anthony Miller & Apollo GraphQL on 8/12/21
+//
+//  This is a derived work derived from
+//  Starscream(https://github.com/daltoniam/Starscream)
```

Comment:
Are you trying to set up markdown here or is there a space missing? 

(also good work playing by [the rules on the apache 2.0 license](https://tldrlegal.com/license/apache-license-2.0-(apache-2.0)))

Reply:
This is present in all the attribution headers; if fixed here please do the others too.

Reply:
Sure, I can change that.